lib/common/results.c

```diff
@@ -216,6 +216,30 @@ static struct pcmk__rc_info {
       "IPC server is blocked by unauthorized process",
       -pcmk_err_generic,
     },
+    { "pcmk_rc_op_unsatisifed",
+      "Operation unsatisfied",
```

Comment:
Might as well as use "Not applicable under current conditions" to be consistent with the exit code